test/e2e/predictor/test_autoscaling.py

```
@@ -407,6 +407,101 @@ async def test_sklearn_env_update():
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
 
 
+@pytest.mark.raw
+@pytest.mark.asyncio(scope="session")
+async def test_sklearn_env_update():
```

Ruff (0.11.9)

Redefinition of unused `test_sklearn_env_update` from line 317

(F811)